tests/data_generation.py

```diff
@@ -449,23 +662,38 @@ def _geodesic_ivp_belongs_data(
         self,
         connection_args_list,
         space_list,
-        n_t_list,
-        tangent_shapes_list,
+        shape_list,
+        n_points_list,
         belongs_atol=gs.atol,
     ):
+        """Generate data to check that connection geodesics belong to manifold.
+
+        Parameters
+        ----------
+        connection_args_list : list
+            List of argument to pass to constructor of the connection.
+        space_list : list
+            List of manifolds on which the connection is defined.
+        shape_list : list
+            List of shapes for random data to generate.
+        n_points_list : list
+            List of number of times on the geodesics.
+        belongs_atol : float
+            Absolute tolerance for the belongs function.
+        """
         random_data = []
-        for connection_args, space, n_t, tangent_shape in zip(
-            connection_args_list, space_list, n_t_list, tangent_shapes_list
+        for connection_args, space, n_points, shape in zip(
+            connection_args_list, space_list, n_points_list, shape_list
         ):
             initial_point = space.random_point()
             initial_tangent_vec = space.to_tangent(
-                gs.random.normal(size=tangent_shape), initial_point
+                gs.random.normal(size=shape), initial_point
             )
             random_data.append(
                 dict(
                     connection_args=connection_args,
                     space=space,
-                    n_t=n_t,
+                    n_points=n_points,
```

Comment:
I remember changing this. However, the function in parameterizer.py needs to be changed too, as I recall that this function is expecting `n_t` for the moment, and not `n_points`?